app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,40 @@
+require 'image_processing/mini_magick'
+
+class ImageUploader < Shrine
+  # Use MiniMagick to process image versions
+  include ImageProcessing::MiniMagick
+
+  ALLOWED_TYPES = %w[image/jpeg image/png]
+
+  # The remove_attachment plugin allows you to delete attachments through checkboxes on the web form.
+  plugin :remove_attachment
+  # The pretty_location plugin attempts to generate a nicer folder structure for uploaded files.
+  plugin :pretty_location
+  # Allows you to define processing performed for a specific action.
+  plugin :processing
+  # The versions plugin enables your uploader to deal with versions, by allowing you to return a Hash of files when processing.
+  plugin :versions
+  # The validation_helpers plugin provides helper methods for validating attached files.
+  plugin :validation_helpers
+  # The store_dimensions plugin extracts and stores dimensions of the uploaded image using the fastimage gem, which has built-in protection agains image bombs.
```

Comment:
__[Misspell]__

"agains" is a misspelling of "against"


<!-- Comment by [SideCI](https://sideci.com) -->